iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractTckSuiteExhaustivityTest.java

```diff
@@ -0,0 +1,84 @@
+package com.powsybl.iidm.network.tck;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+import org.junit.jupiter.params.ParameterizedTest;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public abstract class AbstractTckSuiteExhaustivityTest {
+
+    @Test
+    void assertAllTckTestsAreInstanced() {
+        ClassLoader classLoader = ClassLoader.getSystemClassLoader();
+        String tckPackageName = "com.powsybl.iidm.network.tck";
+        List<String> tckTestClasses = getPackageClasses(classLoader, tckPackageName)
+                .filter(clazz -> containsTest(classLoader, clazz))
+                .filter(clazz -> !clazz.contains("$"))
+                .collect(toList());
+        assertNotEquals(0, tckTestClasses.size(), "Tck package is not loaded in this build");
+
+        String implementationPackageName = this.getClass().getPackageName();
+        Stream<String> implementationTestClasses = getPackageClasses(classLoader, implementationPackageName);
+
+        List<String> tckTestClassInheritors = implementationTestClasses
+                .map(res -> getAncestorFromPackage(classLoader, tckPackageName, res))
+                .filter(Optional::isPresent).map(Optional::get)
+                .collect(toList());
+
+        List<String> unimplementedTckTestClasses = tckTestClasses.stream()
+                .filter(clazz -> !tckTestClassInheritors.contains(clazz))
+                .collect(toList());
+
+        assertEquals(List.of(), unimplementedTckTestClasses,
```

Comment:
It would be cleaner to use
```suggestion
        assertTrue(unimplementedTckTestClasses.isEmpty(),
```